### DISCUSSION TITLE
How to search where "onetag = anothertag" vs "tag = literal"

### DISCUSSION THREAD

**Discussion by bass0159** [quodlibet/quodlibet#3843] [Q&A] [answered]
I have scoured the QuodLibet docs and searched the internet, but I can't find any examples of how to code a search that will compare one tag to another tag.  All of the examples I find are of the Tag to Literal type (i.e. Artist = Beatles) but I am trying to code a search where one tag equals (or does not equal) another tag (i.e. artist != originalartist).  I find it hard to believe that this capability doesn't exist, so I'm guessing that I'm overlooking something obvious.

**Comment by declension** [comment, marked as answer]
Sorry, missed this one.

You can do all this and more with the _Python Query_ plugin. Enable it first, then use something like:

```@(python: s["artist"] != s["originalartist"] and s["originalartist"])```
